logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/DeleteMessageUseCase.kt

```diff
@@ -32,10 +30,13 @@ class DeleteMessageUseCase(
 
         val generatedMessageUuid = uuid4().toString()
         return clientRepository.currentClientId().flatMap { currentClientId ->
-            val message = Message(
-                id = generatedMessageUuid,
-                content = if (deleteForEveryone) MessageContent.DeleteMessage(messageId) else MessageContent.DeleteForMe(messageId),
-                conversationId = if (deleteForEveryone) conversationId else conversationRepository.getSelfConversationId(),
+                val message = Message(
+                    id = generatedMessageUuid,
+                    content = if (deleteForEveryone) MessageContent.DeleteMessage(messageId) else MessageContent.DeleteForMe(
+                        messageId,
+                        conversationId = conversationId.value
```

Comment:
Here, missing qualifiedId too?

Perhaps we should not make it optional, as it's too easy to forget about it.